tests/jballerina-integration-test/src/test/resources/configurables/configStructuredTypesProject/Config.toml

```diff
@@ -0,0 +1,29 @@
+xmlVar1="<book><name>Sherlock Holmes</name><author>Arthur Conan Doyle</author></book>"
```

Comment:
Why do we need a separate `Config.toml` here? We already have a config file named `Config_xml.toml` for this configuration. Can't we use that?